chain/chain/src/chain.rs

```diff
@@ -3018,6 +3018,36 @@ impl Chain {
             .ok_or_else(|| ErrorKind::DBNotFoundErr(format!("EXECUTION OUTCOME: {}", id)).into())
     }
 
+    /// Retrieve the up to `max_headers_returned` headers on the main chain
+    /// `hashes`: a list of block "locators". This function will find the first block in `hashes`
```

Comment:
s/first/oldest ?

Reply:
It's the same, because it assumes that `hashes` is ordered. I'll add that.